backend/copr_backend/storage.py

```diff
@@ -358,13 +358,15 @@ def delete_builds(self, dirname, chroot_builddirs, build_ids):
             response = self.client.delete_content(repository, list_of_prns)
             task = response.json()["task"]
             response = self.client.wait_for_finished_task(task)
-            resources = response.json()["created_resources"]
-            if resources:
+            if response.ok:
```

Comment:
The task API is always going to return 200 ok. You should check the state of the task. 

`if response.json()["state"] == "completed":`

Reply:
Good point, thank you. I'll check both of them, just to be sure.